jdaviz/configs/default/plugins/collapse/collapse.py

```diff
@@ -170,7 +180,7 @@ def _save_collapsed_spec_to_fits(self, overwrite=False, *args):
                 return
 
         filename = str(filename)
-        CCDData(self.collapsed_spec).write(filename)
```

Comment:
Weird that you don't assign CCDData at creation but only at writing it out. I fixed it.